src/app/frontend/replicasetlist/replicasetlist.scss

```diff
@@ -11,28 +11,30 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.    
 // See the License for the specific language governing permissions and   
 // limitations under the License.    
-   
+
+@import '../../assets/styles/color_variables';
+
 .kd-replicaset-card {        
   width: 400px;    
 }
 
+// This override button style to make it look like a link.
 .kd-replicaset-card-logs {
-  color: inherit;
-  text-decoration: none;
-  /* This override button style to make it look like a link */
-  padding: inherit;
-  margin: inherit;
-  min-width: inherit;
-  min-height: inherit;
-  line-height: 0px;
   border-radius: inherit;
+  color: inherit;
   font-size: inherit;
   font-weight: inherit;
+  line-height: 0;
+  margin: inherit;
+  min-height: inherit;
+  min-width: inherit;
+  padding: inherit;
+  text-decoration: none;
   text-transform: inherit;
-}
 
-.kd-replicaset-card-logs:after {
-  content: '\25BC';
+  :after {
```

Comment:
It works for me:

![Logs arrow](http://i.imgur.com/E4xROqG.png)

Tested on Chrome with `gulp serve:prod`.